source/utilities/realtime/dataman/DumpMan.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+
+#include "DumpMan.h"
+
+std::shared_ptr<adios::realtime::DataManBase> getMan()
+{
+    return std::shared_ptr<adios::realtime::DataManBase>(
+        new adios::realtime::DumpMan);
+}
+
+namespace adios
+{
+namespace realtime
+{
+
+int DumpMan::init(json p_jmsg)
+{
+    if (p_jmsg["dumping"] != NULL)
+    {
+        m_dumping = p_jmsg["dumping"].get<bool>();
```

Comment:
I don't particularly care if we keep dumpman or not, it was just something I could add to test with.